src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -11,6 +11,9 @@
         <li class="show-on-desktop"><a href="{{ url('devhub.addons') }}">
           {{ _('My Add-ons') }}
         </a></li>
+        <li class="show-on-desktop"><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link">
+          {{ _('Extension Workshop') }}
+        </a></li>
```

Comment:
lets move this out of the `{% if ... %}` block so it's shown to logged out users too.